### YamlMime:ManagedReference
items:
- uid: System.ServiceModel.AddressAlreadyInUseException
  id: AddressAlreadyInUseException
  children:
  - System.ServiceModel.AddressAlreadyInUseException.#ctor
  - System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String)
  - System.ServiceModel.AddressAlreadyInUseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: AddressAlreadyInUseException
  nameWithType: AddressAlreadyInUseException
  fullName: System.ServiceModel.AddressAlreadyInUseException
  type: Class
  summary: "Unntaket forårsakes når en adresse er utilgjengelig fordi den allerede er i bruk."
  remarks: "Dette unntaket er avledet fra <xref:System.ServiceModel.CommunicationException>som representerer en type kan gjennopprettes feil som kan bli utløst under kommunikasjon mellom sluttpunkt og som robust [!INCLUDE[indigo1](~/add/includes/ajax-current-ext-md.md)] klienten og tjenesten er forventet å behandle.</xref:System.ServiceModel.CommunicationException> Slik at et behandlingsprogram for <xref:System.ServiceModel.CommunicationException>fra fanger inn mer spesifikke AddressAlreadyInUseException denne innhentingsunntak før behandler <xref:System.ServiceModel.CommunicationException>.</xref:System.ServiceModel.CommunicationException> </xref:System.ServiceModel.CommunicationException>"
  syntax:
    content: 'public class AddressAlreadyInUseException : System.ServiceModel.CommunicationException'
  inheritance:
  - System.ServiceModel.CommunicationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor
  id: '#ctor'
  parent: System.ServiceModel.AddressAlreadyInUseException
  langs:
  - csharp
  name: AddressAlreadyInUseException()
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException()
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException()
  type: Constructor
  assemblies:
  - System.ServiceModel
  namespace: System.ServiceModel
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ServiceModel.AddressAlreadyInUseException&quot;> </xref> klasse."
  syntax:
    content: public AddressAlreadyInUseException ();
    parameters: []
  overload: System.ServiceModel.AddressAlreadyInUseException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.ServiceModel.AddressAlreadyInUseException
  langs:
  - csharp
  name: AddressAlreadyInUseException(String)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(String)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(String)
  type: Constructor
  assemblies:
  - System.ServiceModel
  namespace: System.ServiceModel
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ServiceModel.AddressAlreadyInUseException&quot;> </xref> klassen med en angitt feilmelding."
  remarks: "Bruk denne konstruktøren når du vil sende en feilmelding til brukeren. Innholdet i meldingen-parameteren må være forstå den tiltenkte brukeren."
  syntax:
    content: public AddressAlreadyInUseException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
  overload: System.ServiceModel.AddressAlreadyInUseException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.ServiceModel.AddressAlreadyInUseException
  langs:
  - csharp
  name: AddressAlreadyInUseException(SerializationInfo,StreamingContext)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(SerializationInfo,StreamingContext)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.ServiceModel
  namespace: System.ServiceModel
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ServiceModel.AddressAlreadyInUseException&quot;> </xref> klassen serialisering informasjon og direkteavspilling som ble angitt."
  remarks: "Denne konstruktøren kalles under deserialisering til Bygg opp unntak objektet overføres via en dataflyt."
  syntax:
    content: protected AddressAlreadyInUseException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Den <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> som inneholder alle data som kreves for å serialisere unntaket."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Den <xref:System.Runtime.Serialization.StreamingContext>som angir kilde og mål for dataflyten.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.ServiceModel.AddressAlreadyInUseException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.ServiceModel.AddressAlreadyInUseException
  langs:
  - csharp
  name: AddressAlreadyInUseException(String,Exception)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(String,Exception)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(String,Exception)
  type: Constructor
  assemblies:
  - System.ServiceModel
  namespace: System.ServiceModel
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.ServiceModel.AddressAlreadyInUseException&quot;> </xref> klassen med en angitt feilmelding og en referanse til det interne unntaket er årsaken til unntaket."
  remarks: "Bruk denne konstruktøren når du vil sende en feilmelding og informasjon om et indre unntak frem til brukeren. Innholdet i meldingen-parameteren må være forstå for de aktuelle brukerne.       Et unntak som er registrert som en direkte resultatet av et tidligere unntak kan inneholde en referanse til det tidligere unntaket i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>egenskapen returnerer den samme verdien som er sendt til konstruktøren, eller `null` referanse (`Nothing` i Visual Basic) Hvis den <xref:System.Exception.InnerException%2A>egenskapen leverer ikke internt unntak verdien til konstruktøren.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public AddressAlreadyInUseException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
    - id: innerException
      type: System.Exception
      description: "Den <xref:System.Exception>som førte til gjeldende unntaket bli utløst.</xref:System.Exception>"
  overload: System.ServiceModel.AddressAlreadyInUseException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.ServiceModel.CommunicationException
  isExternal: false
  name: System.ServiceModel.CommunicationException
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor
  parent: System.ServiceModel.AddressAlreadyInUseException
  isExternal: false
  name: AddressAlreadyInUseException()
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException()
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException()
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String)
  parent: System.ServiceModel.AddressAlreadyInUseException
  isExternal: false
  name: AddressAlreadyInUseException(String)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(String)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.ServiceModel.AddressAlreadyInUseException
  isExternal: false
  name: AddressAlreadyInUseException(SerializationInfo,StreamingContext)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(SerializationInfo,StreamingContext)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor(System.String,System.Exception)
  parent: System.ServiceModel.AddressAlreadyInUseException
  isExternal: false
  name: AddressAlreadyInUseException(String,Exception)
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException(String,Exception)
  fullName: System.ServiceModel.AddressAlreadyInUseException.AddressAlreadyInUseException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.ServiceModel.AddressAlreadyInUseException.#ctor*
  parent: System.ServiceModel.AddressAlreadyInUseException
  isExternal: false
  name: AddressAlreadyInUseException
  nameWithType: AddressAlreadyInUseException.AddressAlreadyInUseException
